### PR TITLE
Db profiling

### PR DESCRIPTION
**Pull request by Osso**
Ability to profile queries
They will display at the bottom on the page when profiling is activated via the profiling=t or via the user control panel.

### CODE REVIEW

File: modules/miscutil/lib/dbquery.py [hunk 1]
```diff
@@ -25,69 +25,59 @@
...
-        import MySQLdb as mysqldb
-        mysqldb = pool.manage(mysqldb, use_threadlocal=True)
-        connect = mysqldb.connect
-    except ImportError:
-        CFG_MISCUTIL_SQL_USE_SQLALCHEMY = False
-        from MySQLdb import connect
+    import sqlalchemy.pool as pool
+    import MySQLdb as mysqldb
+    mysqldb = pool.manage(mysqldb, use_threadlocal=True)
+    connect = mysqldb.connect
```
Comment (kaplun) [inline modules/miscutil/lib/dbquery.py:55]: ?? SQLAlchemy is not necessarily installed on master. If the flag is set to True and is not installed then :boom:.

**Comment by jalavik** [comment]
Assigned to @kaplun for final approval and deployment.

File: modules/miscutil/lib/dbquery.py (hunk 1 above)
Comment (Osso) [inline modules/miscutil/lib/dbquery.py:55]: Clearly if you set the FLAG to True, you want the tools and ignoring errors silently is just bad practice.

**Comment by kaplun** [comment]
Superseeded by SQLAlchemy

**Pull request closed without merging**